src/main/java/srijana/workshop_1/Task_2.java

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
Change already exists in PR https://github.com/tpolishchuk/java-core-trainings/pull/14/files, needs to be removed